### PR TITLE
fix(components): remove hover active focus and disabled from workspace

### LINKED ISSUE #2601: Audit and remove pf-m-hover/focus/active workspace styles
Let's audit our use of pf-m-hover/focus/active and where we're using this to mimic `:hover`, `:focus`, and `:active` for demonstrating the states in examples, remove that CSS and update any examples that use the classes.

### PR DESCRIPTION
**Pull request by christiemolloy**
closes #2601 

## Breaking Changes

- will update later

### CODE REVIEW

File: src/patternfly/components/Button/button.scss [hunk 1]
```diff
@@ -430,20 +416,17 @@
...
 
-    &:hover,
-    &.pf-m-hover {
+    &:hover {
       --pf-c-button--m-plain--Color: var(--pf-c-button--m-plain--hover--Color);
       --pf-c-button--m-plain--BackgroundColor: var(--pf-c-button--m-plain--hover--BackgroundColor);
     }
 
-    &:active,
-    &.pf-m-active {
```
Comment (mcoker) [inline src/patternfly/components/Button/button.scss:440]: did you mean to remove this?

Comment (christiemolloy) [reply src/patternfly/components/Button/button.scss:440]: nope!

File: src/site/pages/modifiers.md [hunk 2]
```diff
@@ -28,13 +28,11 @@ title: Modifiers
 - pf-m-flex-3
 - pf-m-flex-4
 - pf-m-flex-5
-- pf-m-focus
```
Comment (mcoker) [inline src/site/pages/modifiers.md:31]: wow, forgot this file existed!

Comment (christiemolloy) [reply src/site/pages/modifiers.md:31]: only found it when i did a search in the repo

Comment (mcoker) [reply src/site/pages/modifiers.md:31]: It creates this page, which I don't know if anyone uses https://pf4.patternfly.org/modifiers

**Comment by christiemolloy** [comment]
@mcoker yep looks good - thanks!

**Pull request merged**